src/cameras/PerspectiveCamera.js

```diff
@@ -99,6 +108,59 @@ class PerspectiveCamera extends Camera {
 
 	}
 
+	/*
+	 * Provides the topRight and bottomLeft corners of the camera's 2D frustum slice at a given distance.
+	 * Results are copied into minTarget and maxTarget input vars.
+	 */
+	getBounds( distance, minTarget, maxTarget ) {
```

Comment:
`.getfrustumBounds()` may be more clear.

You need to make clear the coordinate frame these results are in.

Can you explain how this intermediate method is useful to a user?

Reply:
I think getBounds(distance) is clear, espc in the context of a PerspectiveCamera only method 👌

Agree the description could be better, mention frustum explicitly + provide better coordinate frame context. This should also help alleviate name related confusion, can amend the comment + docs when I get a second.

Perhaps getBounds is not as useful as the two other functions proposed but I didn't see any harm to expose it publicly. Similar to how this PR has built two helper methods ontop of this method, I kinda figured someone else might use this as their own building block for something more bespoke. Should we make it internal?

Reply:
We add methods if there is both user demand and a compelling reason to do so.

It was my understanding there was user demand for the frustum width and frustum height methods. I do agree with @Mugen87 and I support adding those.

Reply:
If we want to move `frustumCorners()` to `CameraUtils`, this method needs to be public.

Renaming the method to `getFrustumBounds()` sounds good to me!

Reply:
Name updated as of https://github.com/mrdoob/three.js/pull/27574/commits/84b0c118b8ee47d513f2a80c343fb57b713a6c9e and comment updated accordingly